srcpkgs/pari/template

```diff
@@ -19,6 +19,7 @@ checksum=81ecf7d70ccdaae230165cff627c9ce2ec297b8f22f9f40742279d85f86dfcb1
 
 build_options="x11"
 build_options_default="x11"
+CFLAGS=-flto
```

Comment:
Adding a comment as to why is important, tho :)

Reply:
I added a comment in template.